org-gtd.el

```diff
@@ -201,11 +203,10 @@ It's suggested that you categorize the items in here somehow, such as:
 ;;;; Customization
 
 (defgroup org-gtd nil
-  "Customize the org-gtd package."
-  :version 0.1
-  :group 'emacs)
+  "An implementation of GTD."
+  :group 'org)
 
-(defcustom org-gtd-directory "~/gtd/"
+(defcustom org-gtd-directory "~/gtd"
```

Comment:
Similarly - you tested this? I _think_ this should work because I'm using `f` for path concatenation, but .. Better safe than sorry :)

Reply:
I've not created a test if that is what you mean. I'm still very much learning things. I was simply following the definition for `org-directory`.